messages/dashboard.json

```diff
@@ -1,4 +1,28 @@
 [
+  {
+    "message": "Enjoying rotki? Consider ",
+    "message_highlight": "supporting us in",
+    "action": {
+      "text": "Gitcoin dApps and Apps round",
+      "url": "https://explorer.gitcoin.co/#/round/42161/608/2"
+    },
+    "period": {
+      "start": 1698062400,
+      "end": 1699315140
```

Comment:
epochs seems to be in `2023`

![image](https://github.com/user-attachments/assets/096b8b74-4f73-418a-8327-bed384dce706)